container/Cargo.toml

```diff
@@ -9,4 +9,5 @@ lazy_static = "1.4.0"
 oci-spec = "0.5.3"
 unshare = { git = "https://github.com/virt-do/unshare", branch = "main" }
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
\ No newline at end of file
+serde_json = "1.0"
+proc-mounts = "0.3.0"
```

Comment:
Add this dependency under `[dev-dependencies]` so that we don't pull it when generating the binary, but only for the tests.

Reply:
Ok, I did it!